blog.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Page</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Neuton:ital@0;1&family=Work+Sans:ital,wght@0,300;0,600;1,300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/normalize.css" />
+    <link rel="stylesheet" type="text/css" href="./styles/styles.css">
+  </head>
+  <body>
+    <a href="#main-content" class="hidden">Skip to main content</a>
+    <main>
+      <header class="flex-header">
+        <div class="flex-header-1">
+          <img src="./images/adoptadog-logo.svg" alt="Adoptadog logo">
+          <h2 class="header-headline">Seattle's premiere dog adoption agency</h2>
+        </div>
+        <div class="flex-header-2">
+          <div class="flex-cart">
+            <img class="flex-icon" src="./images/cart-icon.svg" alt="shopping cart icon">
+            <p class="flex-price">$1234.56</p>
+          </div>
+          <a href="./checkout.html">
+            <p class="green-button">Checkout</p>
+          </a>
+        </div>
+      </header>
+      <nav>
+        <ul class="flex-nav">
+          <li class="flex-nav-home"><a href="./index.html">Home &nbsp; &nbsp;</a></li>
+          <li class="flex-nav-dogs"><a href="./dogs.html">Dogs &nbsp; &nbsp;</a></li>
+          <li class="flex-nav-blog"><a href="./blog.html">Blog</a></li>
+        </ul>
+      </nav>
+      <section class="blog-container">
+        <h1 class="grid-blog-heading">Adoptadog Blog</h1>
+        <img class="grid-blog-1" src="./images/blog-1.jpg" alt="Woman and dog sitting on mountaintop with backs to viewer">
```

Comment:
When changing screen sizes, seem to be getting some strange behavior here. I would suggest wrapping each of your blog articles (image, title, text) in a div tag. Then, style these with flexbox. I would not use CSS grid for this. I think you'll find flex a bit easier to work with. Try seeing if you can style your blog posts with this structure:
`
<div class="blog-article">
<img></img>
<div class="blog-text">
<h1></h1>
<p></p>
</div>
</div>
`

Then get your content to be responsive by styling the blog-article tag with flex, and then blog-text with flex to keep the title and paragraph text together.